Python/bytecodes.c

```diff
@@ -3631,8 +3522,6 @@ family(load_attr) = {
     LOAD_ATTR_PROPERTY, LOAD_ATTR_SLOT, LOAD_ATTR_WITH_HINT,
     LOAD_ATTR_METHOD_LAZY_DICT, LOAD_ATTR_METHOD_NO_DICT, LOAD_ATTR_METHOD_WITH_DICT,
     LOAD_ATTR_METHOD_WITH_VALUES };
-family(load_const) = { LOAD_CONST, LOAD_CONST__LOAD_FAST };
-family(load_fast) = { LOAD_FAST, LOAD_FAST__LOAD_CONST, LOAD_FAST__LOAD_FAST };
```

Comment:
Why are these families being removed?

Reply:
This part of the file isn't parsed anyway, and of we moved these families into a part where they *would* be parsed the generator would complain, because it currently doesn't support super-instructions as part of a family.

See also https://github.com/faster-cpython/ideas/issues/495